src/hooks/core/websocket/types.d.ts

```diff
@@ -27,6 +27,7 @@ export type GetstreamMessage = {
   text: string;
   isSystem?: boolean;
   textOwnMessage?: string;
+  reply_data?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The type of `reply_data` is marked as `any`. It's generally a good practice to avoid using `any` in TypeScript because it defeats the purpose of static typing. If possible, replace `any` with a more specific type.
```diff
-  reply_data?: any;
+  reply_data?: ReplyDataType; // Replace ReplyDataType with the actual type you expect.
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->